src/screens/SignIn/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import nimbleLogoWhite from 'assets/images/icons/nimble-logo-white.svg';
+import background from 'assets/images/illustrations/background-sign-in.png';
+import ElevatedButton from 'components/ElevatedButton';
+import TextInput from 'components/TextInput';
+
+export const signInScreenTestIds = {
+  nimbleLogo: 'sign-in__nimble-logo',
+  signInForm: 'sign-in-form',
+  emailLabel: 'sign-in-form__email',
+  emailField: 'sign-in-form__input-email',
+  passwordLabel: 'sign-in-form__password',
+  passwordField: 'sign-in-form__input-password',
+  forgotButton: 'sign-in-form__forgot-button',
+  signInButton: 'sign-in-form__button',
+};
+
+const SignInScreen = (): JSX.Element => {
+  return (
+    <div
+      className="sign-in bg-cover min-h-screen flex flex-col justify-center items-center"
+      style={{ backgroundImage: 'url(' + background + ')' }}
+    >
+      <img className="mx-auto" src={nimbleLogoWhite} alt="nimble logo" data-test-id={signInScreenTestIds.nimbleLogo} />
+      <div className="mt-6 mb-8 text-white opacity-60 text-regular">Sign in to Nimble</div>
+      <form className="w-80" data-test-id={signInScreenTestIds.signInForm}>
+        <div className="mb-6">
+          <TextInput
+            label="Email"
+            labelDataTestId={signInScreenTestIds.emailLabel}
+            inputAttributes={{
+              id: 'form-sign-in-email',
+              required: true,
+              type: 'email',
+              'data-test-id': signInScreenTestIds.emailField,
```

Comment:
```suggestion
              data-test-id: signInScreenTestIds.emailField,
```

Reply:
@nvminhtue unfortunately, I cannot remove the single quotes due to the error 🙏 
![image](https://github.com/manh-t/react-survey/assets/60863885/57605370-8ee6-4b13-9737-78498f290463)